sc/social/like/subscribers.py

```diff
@@ -73,16 +72,12 @@ def social_media_record_synchronizer(event):
         # Plone record modified; synchronize sc.social.like record
         record = ISocialLikeSettings.__identifier__ + '.' + field
     else:
-        logger.debug(u'Schema not being tracked')
+        logger.debug('Schema not being tracked')
         return
 
     registry = getUtility(IRegistry)
     # this will fire the aditional IRecordModifiedEvent
-    try:
-        registry[record] = str(event.record.value)
-    except WrongType:
-        # Plone 5 declares records as TextLine
-        registry[record] = unicode(event.record.value)
+    registry[record] = str(event.record.value)
```

Comment:
```
# Plone 5 declares records as TextLine		
registry[record] = unicode(event.record.value)
```
Funny, this was supposed to make it ok in Plone 5 but removing it the test is still ok https://travis-ci.org/collective/sc.social.like/jobs/385858397#L1725

Reply:
[I fixed that for Plone 5.1](https://github.com/plone/Products.CMFPlone/pull/1967) and I'm dropping support for Plone 5.0; in fact the package is still not compatible with Plone 5 at all, but I want to simplify the code.